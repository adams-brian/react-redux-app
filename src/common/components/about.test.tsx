import { shallow } from 'enzyme';
import * as React from 'react';

import About from './about';

describe("About", () => {
  
  it('renders as expected', () => {
    const component = shallow(<About />);
    expect(component).toMatchSnapshot();
  });

});
