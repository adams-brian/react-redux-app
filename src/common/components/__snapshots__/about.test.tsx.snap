// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders as expected 1`] = `
<div
  className="about-container"
>
  <h1>
    About
  </h1>
  <p>
    A simple app to explore patterns and best practices using React and Redux
  </p>
</div>
`;
